taccsite_cms/templates/standard.html

```diff
@@ -1,4 +1,17 @@
 {% extends "base.html" %}
 {% load cms_tags %}
 
+{% block assets_custom %}
+  {{ block.super }}
+
+  {# TODO: FP-1318: Solve section overflow in `o-section` code #}
+  <style>
+  /* To hide `.o-section--style-…` and `.o-section__banner-image` overflow */
+  /* FAQ: Isolate from other templates to avoid unexpected side-effects */
+  main,
+  /* HACK: Using ID until upstream updates markup via `quick/create-and-document-element-stylesheets` */
+  #content-wrap { overflow: hidden; }
+  </style>
```

Comment:
This fix is already working for Frontera home and can fix UTRC home. See https://github.com/TACC/Core-CMS-Resources/pull/108/files.